src/workerd/api/tests/url-test.js

```diff
@@ -9035,448 +9035,6 @@ export const urlPatternBasics = {
   },
 };
 
-export const urlPatternMdn1 = {
```

Comment:
Would appreciate a review-comment noting why this deletion is OK.
Assuming it's because these tests are duplicates of WPT?

Reply:
Yes, @jasnell raised this concern as well in his previous review.